eigrpd/eigrp_zebra.c

```diff
@@ -93,7 +93,8 @@ static int eigrp_zebra_route_notify_owner(ZAPI_CALLBACK_ARGS)
 	enum zapi_route_notify_owner note;
 	uint32_t table;
 
-	if (!zapi_route_notify_decode(zclient->ibuf, &p, &table, &note))
+	if (!zapi_route_notify_decode(zclient->ibuf, &p, &table, &note,
```

Comment:
Please fix the commit subject message to accurately reflect what is going on here.